meta-iot2000-bsp/recipes-bsp/acpi-upgrades/acpi-upgrades.bb

```diff
@@ -7,14 +7,14 @@ inherit deploy
 
 DEPENDS = "acpica-native"
 
-SRC_URI = "file://iot2000"
+SRC_URI_iot2000 = "file://iot2000"
 
 do_compile() {
 	rm -fr ${WORKDIR}/acpi-upgrades
 
 	install -d ${WORKDIR}/acpi-upgrades/kernel/firmware/acpi
 
-	for table in ${WORKDIR}/${MACHINE}/*.asl; do
+	for table in ${WORKDIR}/*/*.asl; do
```

Comment:
This does not look correct for your use case, an overloaded iot2000 machine. You won't pull the tables then, will you?